frontend/src/views/setting/panel/api-interface/index.vue

```diff
@@ -179,6 +194,7 @@ const onSave = async (formEl: FormInstance | undefined) => {
             apiKey: form.apiKey,
             ipWhiteList: form.ipWhiteList,
             apiInterfaceStatus: form.apiInterfaceStatus,
+            apiKeyValidityTime: form.apiKeyValidityTime,
         };
         loading.value = true;
         await updateApiConfig(param)
```

Comment:
The provided code has a few minor improvements and clarifications, but there are no obvious irregularities or issues. Here is a brief review:

1. **Form Item Labels**: The label for `apiKeyValidityTime` is now in the correct position within the `el-form`.

2. **Template Append Unit**: The unit "minute" correctly appended to the `el-input` for `apiKeyValidityTime`. This helps users understand that time durations should be expressed in minutes.

3. **Initial Values**: The initial value of `apiKeyValidityTime` is set to 120, which makes sense given its default setting in many systems. However, it would be beneficial if you could verify this with appropriate system requirements.

Overall, the changes make the form more user-friendly by providing clear instructions on how to input data and ensuring consistency across similar fields. If there are specific performance optimizations needed based on your application's use case, feel free to ask!